lib/wordmove/deployer/ftp.rb

```diff
@@ -2,7 +2,7 @@ module Wordmove
   module Deployer
     class FTP < Base
       def initialize(environment, options)
-        super
+        super(environment, options)
```

Comment:
Calling `super` without arguments should call its parent initialize method passing every arguments in the child initialize method.

If you use `super()`, instead, you will call the parent initialize method without passing any argument

Source: https://www.rubyguides.com/2018/09/ruby-super-keyword/

Reply:
Got it. BTW If it's ok for you I'd resolve this one w/o any modification.

It does not change anything, that's true, but I feel it more readable for people with less ruby black-implicit-fucking-magic knowledge.

Reply:
Ups...you're off today...I'll cheat w/o waiting for your thumbs up ;) Sorry :P